### PR TITLE
Update changelog for v1.2.0 release

### PR DESCRIPTION
**Pull request by wittejm**
Excludes the fresh pile of refactors. This is waiting on #698

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+- Show the arrest date for both dismissals and convictions (#683)
+- Show the time analysis even if there is an error due to open cases (open cases are ignored in the analysis) (#649)
+- Show the "type" of charge as it is understood for the purposes expungement logic (#698, pending)
+- Minor display format fixes (#669)
+
+## Expungement Logic Fixes
+
+- Correctly identify violation charges (#668)
+- Class B Felony convictions are ineligible if there are any subsequent charges (#653)
+- Class B Felony convictions follow the usual rules for affecting other charges' time eligibility
```
Comment (KentShikama) [inline Changelog.md]: This is missing a PR num

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+
+- Display error message in the results if the record has open cases or missing/unrecognized dispositions (#685, #686)
+- Show the arrest date for both dismissals and convictions (#683)
+- Show the time analysis even if there is an error due to open cases (open cases are ignored in the analysis) (#649)
+- Show the "type" of charge as it is understood for the purposes expungement logic (#698, pending)
+- Minor display format fixes (#669)
+
+## Expungement Logic Fixes
+
+- Correctly identify violation charges (#668)
```
Comment (KentShikama) [inline Changelog.md]: Why not just use the PR title here "More robust string comparison to identify violations in expunger"?

Comment (wittejm) [reply Changelog.md]: I figured that phrase doesn't mean much to a non-coder.

Comment (KentShikama) [reply Changelog.md]: What about "Make violation charge identification more robust"

**Pull request merged**